cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -231,7 +231,8 @@ private void searchChildContextAndRespond(HttpResponder responder, String contex
     List<TagValue> tagValues = parseTagValues(contextPrefix);
     contextPrefix = toCanonicalContext(tagValues);
 
-    MetricSearchQuery searchQuery = new MetricSearchQuery(0, Integer.MAX_VALUE - 1, 1, -1, humanToTagNames(tagValues));
+    MetricSearchQuery searchQuery = new MetricSearchQuery(0, Integer.MAX_VALUE - 1, Integer.MAX_VALUE, -1,
```

Comment:
same. Can we move Integer.MAX_VALUE into a TOTALS_RESOLUTION and/or in MAX_RESOLUTION constant into MetricStore interface?